### DISCUSSION TITLE
Thank you for making this

### DISCUSSION THREAD

**Discussion by vitalifa** [FlowiseAI/Flowise#177] [General]
This project has made it a lot more fun to prototype and explore complex combinations of LLMs and tools. Such a great interface too, suprisingly well polished.

Also, your videos on YouTube were really helpful. You're a naturally good presenter @HenryHengZJ, you should make more videos.

**Comment by HenryHengZJ** [comment]
wow never thought that im a good presenter in any way 😄 appreciate the kind words!

**Reply by vitalifa** [reply]
When talking to an audience through a camera, most people will either come off like a dead fish or coked out and overly animated. Instead, your delivery is authentic and makes the viewer feel like you're casually talking to them directly. It's very difficult to get that right without a lot of experience, but it comes naturally to some people. Brother, you're one of those people that has authentic delivery built in to them, can hook an audience without any gimmicks or fast-cuts.

**Comment by eghan** [comment]
also came here to say thanks for making and working on this fantastic tool

**Comment by camilordg96** [comment]
yeah guys thanks a lot!!
